packages/babel-plugin-transform-classes/README.md

```diff
@@ -119,3 +119,37 @@ class Bar extends Foo {
 When `Bar.prototype.foo` is defined it triggers the setter on `Foo`. This is a
 case that is very unlikely to appear in production code however it's something
 to keep in mind.
+
+### `builtins`
+
+`Array<string>`, defaults to `[]`.
+
+When extending a native constructor (e.g., `class extends Array {}`), it needs
+to be wrapped.
```

Comment:
A couple of changes I'd reccomend:

 * Change "constructor" to "class" (since the key distinction here is constructors that can't be called with `Function.apply`, which are class constructors) and mention a little about w
 * Clarify what needs to be wrapped: the subclass or the native class
 * Add just a little hint on why the wrapper is necessary 

"When extending a native class (e.g., `class extends Array {}`), the native class needs to be wrapped in order to support Babel's emulation of `super()` calls."